api-specs/Konnect/v2/yaml/portal-management.yaml

```diff
@@ -2667,6 +2667,7 @@ components:
           enum:
             - us
             - eu
+            - au
```

Comment:
Is the au region publicly available yet? If not, we might have some issues with using these automated updates to pull specs into the docs, since we're getting some non-public changes. @mheap @acgoldsmith

Reply:
@lena-larionova It's only available to new customers. It will be available to all customers next week.